Storage.md

```diff
@@ -0,0 +1,209 @@
+This guide will describe how FullScreenShenanigans projects keep track of game information.
+
+### Table of Contents
+1. [ItemsHoldr](#itemsholdr)
+    1. [Local Storage](#local-storage)
+    2. [Items](#items)
+    3. [Updating](#updating)
+    4. [Clearing and Defaults](#clearing-and-defaults)
+    5. [Elements](#elements)
+2. [StateHoldr](#stateholdr)
+
+# ItemsHoldr
+
+ItemsHoldr is a versatile container to store and manipulate values in localStorage, and optionally keep an HTML container to show these values.
+
+## Local Storage
+
+ItemsHoldr keeps track of items and their values while the game is running.
+It can also save this information to `localStorage`.
+Other websites also use `localStorage`, so ItemsHoldr has a prefix property to differentiate which information belongs to which source.
+
+```typescript
+let ItemsHolder: ItemsHoldr = new ItemsHoldr({
+    prefix: "FullScreenShenanigans"
+});
+```
+
+## Items
+
+To add, update, and remove an item in the collection, use `addItem`, `setItem`, and `removeItem` respectively.
+
+```typescript
+ItemsHolder.addItem("color", { value: "red" });
+ItemsHolder.setItem("color", "purple");
+ItemsHolder.removeItem("color");
+```
+
+To manually update a changed item's value in localStorage, use `saveItem`.
+To save all the items' values to localStorage, use `saveAll`.
+
+```typescript
+ItemsHolder.saveItem("color");
+ItemsHolder.saveAll();
+```
```

Comment:
It feels weird having half the docs for saving up here and half down below. How about you move these to just above the Auto Save section.